tests/unittests/gradCheckTest.cpp

```diff
@@ -199,11 +200,29 @@ static void gradientCheckGroupConv(size_t numInputChan, size_t group) {
   performGradCheck(IP, result, A, Ex, &inputs, &outputs, 0.001, 0.004);
 }
 
-TEST(Network, gradientCheckConv) { gradientCheckGroupConv(1, 1); }
+TEST(Network, gradientCheckConv) {
```

Comment:
can you use a similar technique as in operatorTest when we support multiple backends for tests?